frontend/main.go

```diff
@@ -78,10 +90,31 @@ func main() {
 		config.Listen = *bindaddr
 	}
 
+	if *splaytreeenabled {

```

Comment:
These ifs are here because of command-line override. first the flag package tries to populate the variable from the `enable-` flags. If the var is true, it will just override the fields in the config.